utils/utils.go

```diff
@@ -54,12 +54,12 @@ func MustGetStringFromEnv(enVarName string) string {
 	return envValue
 }
 
-func GetMultipliesStringsFromEnvDefault(envVarName string, defaultValue []string) []string {
+func GetMultipleStringsFromEncDefault(envVarName string, defaultValue []string) []string {
```

Comment:
Typo (+ `WithDefault`):
```suggestion
func GetMultipleStringsFromEnvWithDefault(envVarName string, defaultValue []string) []string {
```